File: dhali/scripts/move.r
#!/usr/bin/env Rscript
library(dplyr)
library(data.table)

# Read each CSV file into a data frame
df <- read.csv("masterArchive-clean.csv", header = TRUE, sep = ",")

new_df <- relocate(df, Unsubscribe.Date..UTC.05., .after = Subscribe.Date..UTC.05.)

# Write to file
fwrite(new_df, "new-masterArchive-clean.csv")

File: dhali/scripts/move.py
#!/usr/bin/env python

# import
import pandas as pd

in_file = "master-archive-clean.csv"
out_file = "new-master-archive-clean.csv"

# Load the CSV file into a DataFrame
df = pd.read_csv(in_file, dtype=str, on_bad_lines="skip")

# get col

# write to file
df.to_csv(out_file, index=False)
